openedx/core/djangoapps/content/learning_sequences/api/processors/enrollment_track_partition_groups.py

```diff
@@ -44,7 +44,7 @@ def load_data(self, full_course_outline) -> None:
             self.user,
             partition_dict_key='id'
         )
-        self.user_group = self.enrollment_track_groups.get(ENROLLMENT_TRACK_PARTITION_ID)
+        self.user_group = self.enrollment_track_groups.get(ENROLLMENT_TRACK_PARTITION_ID)  # type: ignore
```

Comment:
What `mypy` error were you seeing here? Maybe we can fix it without ignoring the types altogether.

Reply:
@mumarkhan999

Reply:
@kdmccormick 
This was the error
<img width="961" alt="image" src="https://user-images.githubusercontent.com/42294172/173036528-c2283272-7acb-4f35-bd7e-dbd5d1ca72ba.png">

Reply:
Thanks. Let's keep the `#type: ignore` in place for now, since I think this might be complicated to fix.

I created a [follow-up ticket](https://github.com/openedx/tcril-engineering/issues/313); could you link to it here?
```suggestion
        # TODO: fix type annotation: https://github.com/openedx/tcril-engineering/issues/313
        self.user_group = self.enrollment_track_groups.get(ENROLLMENT_TRACK_PARTITION_ID)  # type: ignore
```